.github/workflows/stale.yaml

```diff
@@ -0,0 +1,28 @@
+name: Stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "21 4 * * *"
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v8
+      with:
+        stale-issue-message: 'This issue has become stale and will be closed automatically within 90 days if no activity is recorded.`
+        stale-pr-message: 'This pull request has become stale and will be closed automatically within 90 days if no activity is recorded.'
+        stale-issue-label: 'life-cycle/stale'
+        stale-pr-label: 'life-cycle/stale'
+        days-before-stale: 90
+        close-issue-message: 'This issue was automatically closed due to inactivity.'
+        close-pr-message: 'This pull request was automatically closed due to inactivity.' 
+        close-issue-label: 'life-cycle/rotten'
```

Comment:
Is `rotten` commonly used in other projects? It has some negative connotations.

Reply:
rotten is used in k8s and k8s-sigs, but we could use another label. abandoned  maybe?

Reply:
I'm happy to align with k8s sigs.